Objects/regions.c

```diff
@@ -482,91 +482,91 @@ PyObject* _Py_MakeImmutable(PyObject* obj)
     return obj;
 }
 
-typedef struct RegionObject RegionObject;
-typedef struct RegionMetadata RegionMetadata;
+typedef struct PyRegionObject PyRegionObject;
+typedef struct regionmetadata regionmetadata;
 
-struct RegionMetadata {
+struct regionmetadata {
     int lrc;  // Integer field for "local reference count"
     int osc;  // Integer field for "open subregion count"
     int is_open;
-    RegionMetadata* parent;
-    RegionObject* bridge;
+    regionmetadata* parent;
+    PyRegionObject* bridge;
 };
 
-struct RegionObject {
+struct PyRegionObject {
     PyObject_HEAD
-    RegionMetadata* metadata;
+    regionmetadata* metadata;
     PyObject *name;   // Optional string field for "name"
 };
 
-static void RegionMetadata_inc_lrc(RegionMetadata* data) {
+static void RegionMetadata_inc_lrc(regionmetadata* data) {
     data->lrc += 1;
 }
 
-static void RegionMetadata_dec_lrc(RegionMetadata* data) {
+static void RegionMetadata_dec_lrc(regionmetadata* data) {
     data->lrc -= 1;
 }
 
-static void RegionMetadata_inc_osc(RegionMetadata* data) {
+static void RegionMetadata_inc_osc(regionmetadata* data) {
     data->osc += 1;
 }
 
-static void RegionMetadata_dec_osc(RegionMetadata* data) {
+static void RegionMetadata_dec_osc(regionmetadata* data) {
     data->osc -= 1;
 }
 
-static void RegionMetadata_open(RegionMetadata* data) {
+static void RegionMetadata_open(regionmetadata* data) {
     data->is_open = 1;
 }
 
-static void RegionMetadata_close(RegionMetadata* data) {
+static void RegionMetadata_close(regionmetadata* data) {
     data->is_open = 0;
 }
 
-static bool RegionMetadata_is_open(RegionMetadata* data) {
+static bool RegionMetadata_is_open(regionmetadata* data) {
     return data->is_open == 0;
 }
 
-static void RegionMetadata_set_parent(RegionMetadata* data, RegionMetadata* parent) {
+static void RegionMetadata_set_parent(regionmetadata* data, regionmetadata* parent) {
     data->parent = parent;
 }
 
-static bool RegionMetadata_has_parent(RegionMetadata* data) {
+static bool RegionMetadata_has_parent(regionmetadata* data) {
     return data->parent != NULL;
 }
 
-static RegionMetadata* RegionMetadata_get_parent(RegionMetadata* data) {
+static regionmetadata* RegionMetadata_get_parent(regionmetadata* data) {
     return data->parent;
 }
 
-static void RegionMetadata_unparent(RegionMetadata* data) {
+static void RegionMetadata_unparent(regionmetadata* data) {
     RegionMetadata_set_parent(data, NULL);
 }
 
-static PyObject* RegionMetadata_is_root(RegionMetadata* data) {
+static PyObject* RegionMetadata_is_root(regionmetadata* data) {
     if (RegionMetadata_has_parent(data)) {
         Py_RETURN_TRUE;
     } else {
         Py_RETURN_FALSE;
     }
 }
 
-static RegionMetadata* Region_get_metadata(RegionObject* obj) {
+static regionmetadata* Region_get_metadata(PyRegionObject* obj) {
```

Comment:
I think conventionwise, this should be:
```suggestion
static regionmetadata* region_get_metadata(PyRegionObject* obj) {
```

Though, I have also seen with a `PyRegion_get_metadata`